src/victory-util/collection.js

```diff
@@ -104,6 +104,11 @@ export default {
     (see the tests for more specifics)
   */
   areVictoryPropsEqual(a, b) {
-    return fastDeepEqual(a, b);
+    try {
+      return fastDeepEqual(a, b);
+    } catch (err) {
+      console.warn("VictoryError: fastDeepEqual. https://github.com/FormidableLabs/victory/issues/964"); //eslint-disable-line
```

Comment:
nitpick: we've got a little logging util we can use instead of `console.warn`. (does the same thing but checks whether you're in a production env first)